src/app/shared/services/db/session.service.ts

```diff
@@ -21,12 +21,13 @@ export class SessionService {
 
     private _constructNewSession(profession: Profession): Partial<Session> {
         const expenses: ExpenseItem[] = [
-            {name: $localize`:@@taxes:Taxes`, cashflow: profession.expenses.taxes},
-            {name: $localize`:@@otherPayments:Other payments`, cashflow: profession.expenses.other},
-            {name: $localize`:@@homeMortgage:Home mortgage`, cashflow: profession.expenses.homeMortgage, value: profession.liabilities.homeMortgage, isLiability: true},
-            {name: $localize`:@@schoolLoans:School loans`, cashflow: profession.expenses.schoolLoan, value: profession.liabilities.schoolLoan, isLiability: true},
-            {name: $localize`:@@carLoans:Car loans`, cashflow: profession.expenses.carLoan, value: profession.liabilities.carLoan, isLiability: true},
-            {name: $localize`:@@creditCards:Credit cards`, cashflow: profession.expenses.creditCard, value: profession.liabilities.creditCard, isLiability: true},
+            { name: $localize`:@@carLoans:Car loans`, cashflow: profession.expenses.carLoan, value: profession.liabilities.carLoan, isLiability: true },
```

Comment:
Keep the order of items as current for better input, this order is the same as in the profession card. The Retail debt is at the bottom of the list.